development/build/scripts.js

```diff
@@ -1023,6 +1023,7 @@ async function getEnvironmentVariables({ buildTarget, buildType, version }) {
       environment,
       testing,
     }),
+    PORTFOLIO_URL: ENVIRONMENT.PORTFOLIO_URL || 'https://portfolio.metamask.io',
```

Comment:
`ENVIRONMENT.PORTFOLIO_URL` is always undefined.

To add a new config entry, it must be added in `development/build/config.js`. Then it will be accessible as `config.PORTFOLIO_URL`.

Reply:
Fixed here: https://github.com/MetaMask/metamask-extension/pull/15849